locale/en/about/working-groups.md

```diff
@@ -177,6 +178,29 @@ Each language community maintains its own membership.
 * [nodejs-uk - Ukrainian (Українська)](https://github.com/nodejs/nodejs-uk)
 * [nodejs-vi - Vietnamese (Tiếng Việt)](https://github.com/nodejs/nodejs-vi)
 
+### [Package Maintenance](https://github.com/nodejs/package-maintenance)
+
+Responsibilities include:
+* Building, documenting and evangelizing guidance, tools and processes that make it easier for maintainers to
+  maintain packages and accept help from those who depend on their packages.
+* Management of repositories within the [pkgjs](https://github.com/pkgjs)
+  GitHub organization including but not limited to:
+  * Managing the list of organization owners which supplement the standard
+    Node.js organization owners as outlined in:
+[https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners](https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners)
+  * Overseeing new repositories (creating, moving, removing)
+  * Managing the maintainer teams for all of the repositories.
+  * Contribution policy for repositories
+* Technical direction for the projects within
+  the [pkgjs](https://github.com/pkgjs)  organization
+* Project governance and process (including this policy)
```

Comment:
Not really clear what "(including this policy)" is referring to. I'm guessing maybe this text came out of a different document

Reply:
I agree that just removing that line makes sense to me

Reply:
Sounds good I'll work on getting a PR up to fix the TSC doc this came from @mhdawson